### ISSUE TITLE
ms prefix broken

### ISSUE THREAD

**Issue by natew** [rofrischmann/inline-style-prefix-all#6]
Input:

```js
var prefix = require('inline-style-prefix-all')

prefix({
  msTransition: 'none'
})
```

Output:

```js
{WebkitMsTransition: "none", msTransition: "none"}
```

Desired (I think):

```js
{ MsTransition: 'none', WebkitTransition: 'none', MozTransition: 'none', transition: 'none' }
```

Alternative example would be:

```js
{ WebkitTransition: 'none' }
// => {} // prefixed with all the right ones
```

**Comment by rofrischmann** [comment]
Thanks for reporting this! Actually the output is kind is weird 😅 I'd suggest nothing to be done at all if `msTransition` is passed as one should not use prefixed properties manually, but I see why you'd want to also add the unprefixed in this case. That's something we should discuss perhaps. But `WebkitMs` really needs to be fixed^^ The `moz` is left out as it is only needed until v15, but we might add a check for this to support legacy versions if using a custom build configuration.
I will tackle this issue right tomorrow.

**Comment by rofrischmann** [comment]
fixed with 1.0.5. 
It now prefixes the values but does not add any other prefixes to already prefixed transition properties. Please only use unprefixed props for this case.
